From 70d3585b1b60f6300aef58745d3f060d516ae4b4 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Fri, 27 Dec 2013 15:59:42 +0800
Subject: [PATCH] tests/util: Link with rt if PIGLIT_HAS_POSIX_CLOCK_MONOTONIC

If PIGLIT_HAS_POSIX_CLOCK_MONOTONIC, piglit-util.c will use clock_gettime,
so, make sure to link against -lrt, too.

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
---
 tests/util/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/util/CMakeLists.txt b/tests/util/CMakeLists.txt
index 83391df..df2ea70 100644
--- a/tests/util/CMakeLists.txt
+++ b/tests/util/CMakeLists.txt
@@ -1,5 +1,9 @@
 add_definitions(-DSOURCE_DIR="${piglit_SOURCE_DIR}/")
 
+if(PIGLIT_HAS_POSIX_CLOCK_MONOTONIC)
+    link_libraries(rt)
+endif()
+
 set(UTIL_INCLUDES
 	${CMAKE_CURRENT_BINARY_DIR}
 	${CMAKE_CURRENT_SOURCE_DIR}
-- 
1.8.5.1

